snakesist/exist_client.py

```diff
@@ -161,19 +218,29 @@ def _put_request(self, url: str, data: str) -> bytes:
             url,
             headers={"Content-Type": "application/xml"},
             auth=HTTPBasicAuth(self.user, self.password),
-            data=data
+            data=data.encode("utf-8")
         )
 
         if response.status_code != requests.codes.ok:
             response.raise_for_status()
 
         return response.content
 
+    def _delete_request(self, url: str) -> None:
+        response = requests.delete(
+            url,
+            headers={"Content-Type": "application/xml"},
+            auth=HTTPBasicAuth(self.user, self.password),
+        )
+
+        if response.status_code != requests.codes.ok:
```

Comment:
i learned yesterday the `response.raise_for_status` can be called unconditionally, b/c it does nothing when the status code is `200`.

Reply:
cool. i removed the line.